egs/wsj0-mix/DeepClustering/local/conf.yml

```diff
@@ -1,8 +1,12 @@
 # Filterbank config
 filterbank:
-  n_filters: 512
-  kernel_size: 512
-  stride: 256
+# n_filters: 512
+# kernel_size: 512
+# stride: 256
+ n_filters: 256
+ kernel_size: 256
+ stride: 64
+ log: True # Use log spectra as input
```

Comment:
If the log is taken in the mask network, then it belongs in a mask network config